middleware/csrf/csrf.go

```diff
@@ -100,8 +104,16 @@ func New(config ...Config) fiber.Handler {
 		} else {
 			cfg.Cookie = ConfigDefault.Cookie
 		}
+
+	}
+
+	if cfg.Storage == nil {
+		cfg.Storage = storage.NewMemoryStorage()
+	}
+
+	if cfg.Generator == nil {
+		cfg.Generator = cryptoRandomGenerator{length: 32}
```

Comment:
If you want to make generator customized, copy the [option](https://github.com/gofiber/fiber/blob/master/middleware/requestid/requestid.go#L20-L23) and [init stuff](https://github.com/gofiber/fiber/blob/master/middleware/requestid/requestid.go#L36) from requestid mw is fine.